src/root.css

```diff
@@ -0,0 +1,10 @@
+:root {
```

Comment:
We already have an index.css file. And this file contains all global-scoped styles.
Maybe it would be better to add this styles to index.css instead of creating another one separate file with global-scoped vars? What do you think? )))

Reply:
You're right, I will.